### PR TITLE
Part of #17670: Replace Typography with Text component

### PR DESCRIPTION
**Pull request by dhruvv173**
## Explanation
This PR is a part of [#17670 ](https://github.com/MetaMask/metamask-extension/issues/17670)
Updating Typography to Text ```cancel-speedup-popover.js``` and ```beta-header/index.jss```
ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js
ui/components/app/beta-header/index.js

Couldnt attach the screenshots of cancel-speedup-popover as the file wasnt found on Storybook

## Screenshots/Screencaps
beta-header/index.js

### Before
WhatsApp Image 2023-04-30 at 12 22 09

### After
WhatsApp Image 2023-04-30 at 12 27 49

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by dhruvv173** [comment]
Hey @georgewrmarshall @PeterYinusa
PR successfully passed all the tests, can you guys review it once?
Thanks!

**Review by Karlitos88 — APPROVED** [review]

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js [hunk 1]
```diff
@@ -97,9 +98,9 @@ const CancelSpeedupPopover = () => {
     >
       <AppLoadingSpinner className="cancel-speedup-popover__spinner" />
       <div className="cancel-speedup-popover__wrapper">
-        <Typography
+        <Text
           boxProps={{ alignItems: AlignItems.center, display: DISPLAY.FLEX }}
-          variant={TypographyVariant.H6}
+          variant={TextVariant.bodySm} as="h6"
```
Comment (NidhiKJha) [inline ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js:103]: Update 
```
 <Text
          boxProps={{ alignItems: AlignItems.center, display: DISPLAY.FLEX }}
          variant={TextVariant.bodySm} as="h6"
```

to 
```
 <Text
          alignItems={AlignItems.center}
          display: {DISPLAY.FLEX }
          variant={TextVariant.bodySm} as="h6"
```

**Review by NidhiKJha — COMMENTED** [review]
Really great start! Thanks for your contribution. Left a suggestion. Also, it would be great to get before/after screenshots for `cancel-speed-popover` to ensure that there are no visual differences

File: ui/components/app/beta-header/index.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
+/* eslint-disable prettier/prettier */
```
Comment (georgewrmarshall) [inline ui/components/app/beta-header/index.js:1]: Why was this line added?

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
+/* eslint-disable prettier/prettier */
```
Comment (georgewrmarshall) [inline ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js:1]: Same question here.

**Review by georgewrmarshall — COMMENTED** [review]
Hi @dhruvv173, thanks for your contribution. Left a couple of questions

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js (hunk 1 above)
Comment (dhruvv173) [reply ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js:103]: Hey @NidhiKJha 
Its supposed to be ```display= {DISPLAY.FLEX}``` instead of ```display: {DISPLAY.FLEX}```, right? because the code throws an error otherwise.
Also, I coulnt find any screenshot of ```cancel-speed-popover``` in the storybook components thats why I was not able to add it in the before/after

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js (hunk 3 above)
Comment (dhruvv173) [reply ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js:1]: Those lines were just to disable prettier formatting in vscode, I will remove those lines and commit changes once I get a response from @NidhiKJha

Comment (dhruvv173) [reply ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js]: Hey @georgewrmarshall 
I have removed those lines and made changes as told by @NidhiKJha
Can you please review the code now?
Thanks!

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js (hunk 1 above)
Comment (NidhiKJha) [reply ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js]: Ah! yes, It should be `display= {DISPLAY.FLEX}`. Thanks for updating it

**Comment by NidhiKJha** [comment]
You need to resolve the lint errors for the CI checks to pass. You can run the command `yarn lint:fix` and it should do :)

File: ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js (hunk 1 above)
Comment (dhruvv173) [reply ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js]: made the changes but it failed a few tests due to formatting issues, will recommit now.
